src/utils/data_inserter.rs

```diff
@@ -43,9 +43,9 @@ no_arg_sql_function!(
 pub fn insert_restaurant(
     conn: &SqliteConnection,
     restaurants_data: &JsonRestaurant,
-    version: &Version,
+    version: i32,
 ) -> i32 {
-    let insertable = map_restaurant_json2insert(restaurants_data, version.id);
+    let insertable = map_restaurant_json2insert(restaurants_data, version);
```

Comment:
I don't like this change, the reason that it initially used a Version reference was to have some enforcement that we're working with a correct version, instead of just giving a random integer.

Even if this was done for performance I would still disagree as this code doesn't need to be performant, as its rarely run and the cost of a reference is so small compared to the SQL queries.

The same goes for the similar changes

Reply:
I don't see the need for needing more than the number from the database.
The previous code fetched selected a whole row of the `version_history`-table. The only property used in the code was the id of the row.
If the code used other properties than the id, then I would agree with you, but currently I do not see any need for sending entire structs.

Reply:
The reason the entire struct is sent is to use the type safety of rust to ensure only versions are given the function. This was the primary reason why I designed it this way.

But if you're still against this then just keep the change. I would be fine with that as the its possible to circumventing this by just initializing a Version struct, weakening my point significantly.